File: Builder/Pizza.java
public class Pizza {

    private String masa = "";
    private String relleno = "";
    private String salsa = "";


    public void setMasa(String masa)
    {
        this.masa = masa;
    }

    public void setRelleno(String relleno)
    {
        this.relleno = relleno;
    }

    public void setSalsa(String salsa)
    {
        this.salsa = salsa;
    }
    
    public void dataPizza()
    {
        System.out.println("");
        System.out.print("Pizza Hawaiana\n\n" + "Tipo masa: " + this.masa + "\nTipo Salsa: " + this.salsa + "\nTipo de relleno: " + this.relleno);
    }
}

File: Builder/Python/Pizza.py
class Pizza:

    _masa = " "
    _relleno = " "
    _salsa = " "
    
    def setMasa(self, masa):
        self._masa = masa
    
    def setRelleno(self, relleno):
        self._relleno = relleno
    
    def setSalsa(self,salsa):
        self._salsa = salsa

    def dataPizza(self):
        print("Pizza: " + self._masa + self._relleno + self._salsa)

